### ISSUE TITLE
PS3 gets really slow when Manual fan speed is enabled

### ISSUE THREAD

**Issue by floppyD** [aldostools/webMAN-MOD#275]
It gets really slow, taking up to 2 minutes to boot and more than 10 minutes to get past the load screen of games that would otherwise load in seconds, and not running other games like The Last of Us at all. Not to mention that XMB gets really laggy as well.
Changing the fan settings to SYSCON solves the issue instantly.
This wasn't happening in previous versions (I don't recall which version exactly).

System:
PS3 Super Slim - 4.84 HFW (HEN 2.32)
webMAN MOD 1.47.25

The issue happens when these exactly settings are set:
![Capture](https://user-images.githubusercontent.com/29208110/66258437-a82c9b00-e77b-11e9-8940-4557b67a3736.JPG)

- P. S: I did some tests with Auto #2 and "Auto in XX°C", there was no issue, so maybe it's related to Manual fan speeds.
- P. S 2: Yep, only Manual and it only gets bad after you reboot the console.

**Comment by J013k** [comment]
Does this problem still exists with HEN 3.0.0 on 1.47.25.10.
Did you tried to uninstall webMAN MOD from setup (Uninstall webMAN MOD).
Install it again?

**Comment by Irome** [comment]
I had the same problem with the latest versions of Webman MOD.
I couldn't control the fan speed that was set really low (at 20%) and after a few minutes I turned on the console, the CPU/RSX temperature rose to 80° compared to the initial 45°.

I solved by installing Webman MOD version 1.47.21.

**Comment by floppyD** [comment]
Sorry for the delay, I really forgot about this issue. Just tried running some games on the latest version (1.47.27) with Manual fan control and didn't notice any issues, XMB seems fine too.  I'll test some games later, but looks like the issue was solved.

**Comment by floppyD** [comment]
Nope, still present. GTA V has major pop in issues, The Last Of Us takes ages to load (it's so slow that instead of just that regular fading sand loadscreen, a "LOADING x.xx%" appears at the bottom right, and it starts increasing really slowly, like 0.5% per second, and keeps resetting all the time). I won't even try to measure how long it takes, but it's already way over 5 minutes and the thing is still at 43.17%. Switching from Manual to SYSCON makes the game load in exactly 10 seconds. Both are BD games, if it helps.

**Comment by floppyD** [comment]
Okay, it seems to be kinda random, but there's definitely a speed penalty by using Manual. Every single time I load TLoU while using SYSCON it takes exactly 10s, but with Manual, it may take 14-15s. When enabling HEN, XMB will get stuck for 5s before letting you move, which doesn't happen at all with SYSCON. I couldn't reproduce the absurd load times after doing some runs.
Just for the record, PS3 HEN 3.0.1 - webMAN 1.47.27

**Comment by Joonie86** [comment]
No further feedback
